(ForumsTeam) Broken Hyperlink

v-viksur opened the pull request:
In this documentation under the section "Monitoring your storage service" "Receive Alert Notifications" hyperlink has broken. I am requesting you to review and update the hyperlink.

rmca14 commented:
@mimig1

mimig1 commented:
@AndrewHoh please remove the keywords. I'm only adding those to articles after I've done the related SEO work on them.

AndrewHoh commented:
@mimig1  done

mimig1 commented:
Also, resolve/address this bug. http://sqlbuvsts01:8080/Main/DocumentDB/_workitems#_a=edit&id=5818039&triage=true

AndrewHoh commented:
Merge conflicts resolved in other PR.

v-viksur committed:
(ForumsTeam) Broken Link

In this documentation under the section "Monitoring your storage service" "Receive Alert Notifications" hyperlink has broken. I am requesting you to review and update the hyperlink.
